TypeCobol/Compiler/CompilationProject.cs

```diff
@@ -83,8 +83,8 @@ public static CompilationProject Load(string projectName, string rootDirectory)
         public EndOfLineDelimiter EndOfLineDelimiter { get; private set; }
         public int FixedLineLength { get; private set; }
         public ColumnsLayout ColumnsLayout { get; private set; }
-        public TypeCobolOptions CompilationOptions { get; private set; }
-        public IAnalyzerProvider AnalyzerProvider { get; private set; }
+        public TypeCobolOptions CompilationOptions { get; set; }
+        public IAnalyzerProvider AnalyzerProvider { get; set; }
```

Comment:
Changing the value of this two properties work only if we discard current FileCompilers and recreate them. So it is a bit misleading. Could you add a comment to not forget that ?

Reply:
Done [4102038](https://github.com/TypeCobolTeam/TypeCobol/pull/2145/commits/4102038a30a2c65ba3e94f71551a3ad68c37f746)